src/WebEid.Security.Tests/Certificate/CertificateDataTest.cs

```diff
@@ -11,7 +9,7 @@ public class CertificateTests
         [Test]
         public void WhenOrganizationCertificateThenSubjectCNAndIdCodeAndCountryCodeExtractionSucceeds()
         {
-            X509Certificate organizationCert = Certificates.GetOrganizationCert();
```

Comment:
Expicit type would be preferrable since it is not inherently clear from the method, what type of certificate is returned.